### DISCUSSION TITLE
question: to decision tree plot

### DISCUSSION THREAD

**Discussion by Arnold1** [pixijs/pixijs#8793] [Q&A]
Hi,

I have a questions. I would like to render a decision tree structure like that, which I could provide with graphviz dot format: https://mljar.com/blog/visualize-decision-tree/output_15_0.svg

I already looked at d3  but when I try to display 25k nodes it really slow with svg: https://jsfiddle.net/k6zupbhx/1/

so I thought I might be able to use pixijs and plot a decision tree. is that possible? do you have any examples?

**Comment by domis86** [comment]
If you display all of "25k nodes" with full details always then probably it will be slow in any framework - no matter if it is d3 or pixi.js

You need to implement techniques like culling (display only those objects which are visible to user) and something to display objects with less details the more zoomed out the view is (if you look at something IRL from 1 km distance then you see only general shape - no small details etc).
